homeassistant/components/camera/bloomsky.py

```diff
@@ -0,0 +1,64 @@
+"""
+homeassistant.components.sensor.bloomsky
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Support for BloomSky weather station.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/bloomsky/
+"""
+import logging
+import requests
+import homeassistant.components.bloomsky as bloomsky
+from homeassistant.components.camera import Camera
+
+DEPENDENCIES = ["bloomsky"]
+
+
+# pylint: disable=unused-argument
+def setup_platform(hass, config, add_devices_callback, discovery_info=None):
+    """ set up access to the BloomSky's camera """
+    for device_key in bloomsky.BLOOMSKY.devices:
```

Comment:
Instead of having to configure each device that is a camera, would it be possible to auto detect camera devices based on a JSON property that they have?

Also, you're missing a great opportunity to use a list comprehension!

```python
add_devices_callback(device for device in bloomsky.BLOOMSKY.devices.values()
                     if device["DeviceName"] in config["devices"])
```